packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/wdio-testing/About.tool.mdx

```diff
@@ -34,3 +60,18 @@ it('should override a rule configuration', () => {
   Terra.validates.accessibility(options);
 });
 ```
+
+## Input Caret
+
+### Terra.hideInputCaret
+
+An editable text field in focus will have a blinking caret. Often times this blinking caret causes inconsistent test failures due to the blinking of the caret during screenshot capture. This situation can be avoided by using `Terra.hideInputCaret` to set the CSS caret color to of the element to be transparent. This method must be placed in a `before`, `beforeEach`, or `it` block or it will not be ran. This method accepts a selector string as an argument and will only apply to the first selector if multiple are found.
```

Comment:
We should mention we will run this automatically on page load for body and is likely handled already

Reply:
Updated: https://github.com/cerner/terra-toolkit/pull/438/commits/825dac196d8e70c66767ce89f6479d4e73a75951